CHANGES.txt

```diff
@@ -1,6 +1,73 @@
 Enable CHANGELOG
 ================
 
+Enable 5.2.0
+============
+
+Thanks to:
+
+* 
```

Comment:
Did you want to fill these in before merging the PR?

Reply:
🤦 yep, whoops